pom.xml

```diff
@@ -224,7 +236,7 @@
               </transformers>
               <!-- This fixes the issue "Invalid signature file digest for Manifest main attributes"
                    cf. http://zhentao-li.blogspot.com/2012/06/maven-shade-plugin-invalid-signature.html -->
-              <filters>
+              <!--filters>
```

Comment:
is this change intentional?

Reply:
Yes, commented out intentionally - not sure its effect... so punting for now, will circle back later to figure out implications.